templates/index.html

```diff
@@ -6,9 +6,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="">
     <meta name="author" content="">
-    <link rel="icon" href="../../favicon.ico">
+    <link rel="icon" href="../resources/favicon.ico">
```

Comment:
I figure at some point we'll have images or pictures, so perhaps we can create a directory called 'resources' for this.